### PR TITLE
WX-1259 Cromwell resource monitoring workflow options

### PR DESCRIPTION
**Pull request by aednichols**
Ticket: [WX-1259](https://broadworkbench.atlassian.net/browse/WX-1259)

Tested on dev Cromwell, verified by examining `WORKFLOW_OPTIONS` column in the `WORKFLOW_STORE_ENTRY` table.

Screenshot 2023-11-03 at 12 02 50

---

**PR checklist**

- [ ] Include the JIRA issue number in the PR description and title
- [ ] Make sure Swagger is updated if API changes
  - [ ] **...and Orchestration's Swagger too!**
- [ ] If you changed anything in `model/`, then you should [publish a new official `rawls-model`](https://github.com/broadinstitute/rawls/blob/develop/README.md#publish-rawls-model) and update `rawls-model` in [Orchestration's dependencies](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/project/Dependencies.scala).
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green, including CI tests
- [ ] **Squash commits and merge** to develop (branches are automatically deleted after merging)
- [ ] Inform other teams of any substantial changes via Slack and/or email

[WX-1259]: https://broadworkbench.atlassian.net/browse/WX-1259?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: core/src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -4820,6 +4829,15 @@ components:
           type: boolean
           default: false
           description: Whether or not to create output columns if they would be empty and optional.  Defaults to false (columns are created).
+        monitoringScript:
+          type: string
+          description: Location of user-defined monitoring script that runs inside the task container
+        monitoringImage:
+          type: string
+          description: User-defined monitoring container that runs as a sibling to the task container
```
Comment (jgainerdewar) [inline core/src/main/resources/swagger/api-docs.yaml]: Might want to state directly that we expect either this or `monitoringScript`, not both.

File: model/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala [hunk 2]
```diff
@@ -414,7 +430,7 @@ trait ExecutionJsonSupport extends JsonSupport {
 
   implicit val ExecutionServiceLogsFormat = jsonFormat2(ExecutionServiceLogs)
 
-  implicit val ExecutionServiceWorkflowOptionsFormat = jsonFormat15(ExecutionServiceWorkflowOptions)
+  implicit val ExecutionServiceWorkflowOptionsFormat = jsonFormat18(ExecutionServiceWorkflowOptions)
```
Comment (jgainerdewar) [inline model/src/main/scala/org/broadinstitute/dsde/rawls/model/ExecutionModel.scala:433]: 15+3=18, math approved ✅

**Review by jgainerdewar — APPROVED** [review]

**Review by JVThomas — APPROVED** [review]

File: core/src/main/resources/swagger/api-docs.yaml (hunk 1 above)
Comment (aednichols) [reply core/src/main/resources/swagger/api-docs.yaml]: We technically don't care what users do, they could monitor multiple ways at once. But definitely clarified 👍

**Comment by aednichols** [comment]
jenkins retest

**Pull request merged**